### ISSUE TITLE
Feature Request: continuously scroll up and down multiple websites

### ISSUE THREAD

**Issue by MarvinMynx** [jpablobr/simple-auto-scroll#16]
Hey,

I found your Chrome add-on “Simple Auto Scrolö” it’s excellent. 
But something I need is missing.
But I can’t find any add-on that can do what I need exactly.
Just a question, do you know another add-on or do you know how to implement the missing function in your addon?
1: I want/need the ability to scroll down tabs while I’m in fullscreen mode 
2: and the ability to scroll up again after “wait at the bottom” seconds is over.
3: Further, I need to set a “wait at the top” timer just the same as “wait time at the bottom” does just for the top of the page.
4: Reload the next tab before it is opening.

Basically, I need to continuously scroll multiple websites up and down while switching to the next tabs after XX seconds over and over again, in Fullscreen, with or without website reload before turning to the next tab.
It’s for a Monitoring TV at the Office. I just tried every single auto scroll addon in the chrome store.
Just Cycle through the tabs or scroll down a page is simple. Nearly every scroll addon in the chrome store does that. But no addon does what I explained above.

Kind Regards
Marvin van Thiel

**Comment by ELD-France** [comment]
Hello, I have the same demands. I’d like to present sports results continuously by a html page that moves up/down/up/down/... continuously like an elevator. Short stop at each end.

**Comment by Aasemoon** [comment]
I have a similar request, perhaps a somewhat simpler one. I'd just like to be able to have the auto scroll continue running on one browser window / tab while I work on another window / tab, without the scroller stopping.
Cheers!
